packages/e2e/test/web-extension/extension/ui.ts

```diff
@@ -126,7 +129,7 @@ const createWallet = async (accountIndex: number) => {
     logger
   });
 
-  // TODO: remove observableWalletName here
```

Comment:
@mkazlauskas , I left the observableWalletName because it will be the user configured name. However, it is not used as unique ID anymore.

Reply:
Maybe let's rename it to better reflect the purpose. How about `friendlyName`?

Reply:
For the moment, let's keep observableWalletName as a way of identifying the wallets in the logs. The walletId is calculated internally and may be difficult to follow logs if are coming from wallets on the same network.